src/components/Activity/Activity.js

```diff
@@ -12,6 +12,7 @@ import { dimensions } from '../Shared/Dimensions';
 import Connected from './Connected';
 import NewRequests from './NewRequests';
 import ScrollableTabView from 'react-native-scrollable-tab-view';
+import DefaultTabBar from './DefaultTabBar';
```

Comment:
I don't think it is necessary if the purpose is for finding import statement more easily